### PR TITLE
Ocular desc rewrite

### PR DESCRIPTION
**Pull request by NWeber-SL**
Review specifically:

* Rewrote 1st paragraph. 
* New bullet on Extensibility
* Combined bullets on interactive and non-interactive modes, and rewrote 

Changed "Pimp my Library" to "Enhance my Library"

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -6,7 +6,9 @@
   * [ShiftLeft Products](introduction/products.md)
   * [ShiftLeft Compliance and Support](introduction/support.md)
   * [ShiftLeft Requirements](introduction/requirements.md)
-* [Using ShiftLeft Ocular](using-ocular/ocular-desc.md)
+* Using ShiftLeft Ocular
```
Comment (vlad-alexandru-ionescu) [inline SUMMARY.md]: This will change the entire URL structure for Ocular docs. All links would have to be updated. What's the motivation behind this change?

File: SUMMARY.md [hunk 2]
```diff
@@ -6,7 +6,9 @@
...
+  * Getting Started
+    * [About ShiftLeft Ocular](using-ocular/getting-started/ocular-desc.md)
```
Comment (vlad-alexandru-ionescu) [inline SUMMARY.md]: I don't know if this makes sense - there's a `Getting Started` section, yet also a `Quick Start` page outside of it.

File: using-ocular/getting-started/ocular-desc.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+You can download a [trial version (Java-only)](https://go.shiftleft.io/ocular-free-trial) of ShiftLeft Ocular. The trial  version has a limited feature set.
+
+ShiftLeft Ocular provides the following core features:
+
+* **Generation of Code Property Graphs (CPG).** (Included in the ShiftLeft Ocular trial version). Based on your application code, ShiftLeft Ocular generates a CPG that can be used as the basis of custom static analysis. CPGs provide an extensible, multi-layered representation of each unique code version. 
+
+* **Extensibility**. ShiftLeft Ocular is highly extensible through the use of custom queries and [Policies](../tutorials/sp.md), by [modifying the OQL language](../tutorials/sp.md#customizing-the-dsl-using-the-extend-my-library-pattern), and by [creating a CPG pass](https://ocular.shiftleft.io/api/io/shiftleft/passes/index.html) and introducing new layers into the CPG.
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/getting-started/ocular-desc.md]: The word `Policies` is linking to SP page. That doesn't make sense.

File: SUMMARY.md (hunk 1 above)
Comment (NWeber-SL) [reply SUMMARY.md]: Discussed this yesterday. Motivation is that as the doc set grows and more articles are added, they need to be grouped or the TOC will become too long and useless. Also as I am learning more about the product, and am understanding how to organize the information.

File: SUMMARY.md (hunk 2 above)
Comment (NWeber-SL) [reply SUMMARY.md]: The Quick Start, and all other conceptual topics, are moved into the Getting Started section in different PRs, and I write and rewrite that material.

File: using-ocular/getting-started/ocular-desc.md (hunk 3 above)
Comment (NWeber-SL) [reply using-ocular/getting-started/ocular-desc.md]: Good catch. I still get mixed up sometimes between policies and profiles. Fixed

File: SUMMARY.md (hunk 1 above)
Comment (vlad-alexandru-ionescu) [reply SUMMARY.md]: We should put together a final version of the ToC, have everyone agree on it and then only change it rarely. Could we skip making ToC change for now and do it after we agree on a final version?

File: SUMMARY.md (hunk 2 above)
Comment (vlad-alexandru-ionescu) [reply SUMMARY.md]: Got it.
